app/assets/javascripts/errors.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
do we need the .coffee and .scss files that rails generated? i'd request we delete extra/unused files if that's ok with yall